### ISSUE TITLE
On Windows 10 getCpuTemperature () always returns 30.0. Is this correct?

### ISSUE THREAD

**Issue by yososs** [oshi/oshi#341]
On Windows 10 getCpuTemperature () always returns 30.0. Is this correct?

WindowsSensors.java Win32_PerfFormattedData_Counters_ThermalZoneInformation data is used.

303 - 273 = 30

While Windows counters are inaccurate,
It will be confirmed before Open Hardware Monitor. Is not it better to prioritize OHM?

When I changed it as follows, I got the correct temperature.
```
    public WindowsSensors(){
        String cpuIdentifier = WmiUtil.selectStringFrom("root\\OpenHardwareMonitor", "Hardware", "Identifier",
                "WHERE HardwareType=\"CPU\"");
        this.tempIdentifierStr = cpuIdentifier.length() > 0 ? cpuIdentifier : null;
    }
```

**Comment by dbwiddis** [comment]
I suppose it makes sense to prioritize OHM.  I would rather Windows just didn't publish the information at all rather than publish wrong info! 

I'd welcome a PR, as I don't have the ability to test that.

**Comment by yososs** [comment]
I want to accurately grasp the current status before PR,
Please share whether there was an environment where CPU temperature can be obtained correctly in Windows version.

In order to make changes so as not to destroy the environment in which I currently operate.

**Comment by dbwiddis** [comment]
I do not have a physical Windows machine that reports the CPU temperature to OSHI.  However, others submitting Issues on this project have indicated they do.  So it would be unwise to remove the Windows built in WMI checks.

Presently, the system tries all the built in WMI options (only once).  If it succeeds, it continues to use that option; if it fails, it always defaults to trying the OHM option (which it assumes can be turned on or off at will).

A revised version of this would simply reverse this order; try the OHM option first.  If it's present, it uses that information.  If not, it goes through the sequential check of the built in Windows WMI options (remembering the first successful one, if any).

**Comment by yososs** [comment]
OK. I think that I will send PR when I get working time.

**Comment by wowselim** [comment]
This seems to be related to an issue I reported earlier https://github.com/oshi/oshi/issues/266

**Comment by dbwiddis** [comment]
Yep, this is the same issue as #266.  If there is documentation on what the numbers are supposed to mean I will be happy to change the calculation.

**Comment by dbwiddis** [comment]
I've attempted to fix this in #357 .  @yososs can you test that PR?
